docs/07_rfp/_category_.json

```diff
@@ -0,0 +1,9 @@
+{
+  "label": "Requests for Proposals (1)",
```

Comment:
Why the "(1)"?

Reply:
it is showing the count, indicating that there is sth live